Extras/HistoryView/UIKit/COTrackViewController.m

```diff
@@ -107,7 +107,7 @@ - (UIColor *)suggestedColorForNode: (id <COTrackNode>)node
 
 - (UITableViewCell *)makeCellForNode: (id <COTrackNode>)node
 {
-    static NSString *TrackCellIdentifier = @"TrackCell";
+    static const NSString *const TrackCellIdentifier = @"TrackCell";
```

Comment:
I made this const mainly because of that `static` there made me think of `FOUNDATION_EXPORT` lines with `static const NSString *const` values. But presumably in general Objective-C classes aren't to be made `const` (right?). So should this be `const` or no?